README.md

```diff
@@ -4,27 +4,38 @@ Welcome to the Nix Community infrastructure project. This project holds all the
 
 ## Services
 
-### Community builder - build01.nix-community.org
+### `Community builder` - build01.nix-community.org
 
 We also provide one x86 hetzner build machine as a public remote builder for the nix community, see [here](roles/builder/README.md) for more information.
```

Comment:
Should we mention the aarch64 machine as well?

Reply:
Added aarch64 for the builder and CI systems.